src/components/CheckboxLabel/CheckboxLabel.module.css

```diff
@@ -1,12 +1,5 @@
-/*------------------------------------*\
-    # CHECKBOX LABEL
-\*------------------------------------*/
-
-/**
- * Checkbox medium variant.
- */
```

Comment:
I'm guessing you're in favor of these header blobs being stripped out wholesale :) they are in the ploppable templates, and don't really add much, given how we currently use them. I suspect this came from the BM code?

Reply:
Yeah, they came from BM.

Deleted them because they don't add anything to these examples. But maybe I should add them back and worry about them later.